db/util/random.go

```diff
@@ -0,0 +1,39 @@
+package util
+
+import (
+	"math/rand"
+)
+
+// List of owners names for testing
+var nameList = []string{"John", "Jane", "Joe", "Jill", "Jack"}
+
+// List of currency code for testing
+var currencyList = []string{"USD", "EUR", "CAD", "CNY", "JPY"}
+
+// By default, the random number generate number between [1, 1000]
+func GetRandomInt() int64 {
+	return rand.Int63n(1000) + 1
+}
+
+func GetRandomIntWithRange(min, max int64) int64 {
+	return rand.Int63n(max-min) + min
+}
+
+func GetRandomStringWithLength(length int64) string {
+	var letters = []rune("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ")
+
+	b := make([]rune, length)
+	for i := range b {
+		b[i] = letters[rand.Intn(len(letters))]
+	}
+
+	return string(b)
+}
+
```

Comment:
Add a functioning comment will be better for all newly added methods. Format: `method name` + `comment`  Like this: 
``` suggestion
// GetRandomOwnerName generates a random owner name
```